packages/api/db/migration/20211122153642_organic_history.js

```diff
@@ -7,17 +7,17 @@ exports.up = async function (knex) {
     t.uuid('location_id')
       .references('location_id')
       .inTable('location').notNullable();
-    t.enu('to_state', ['Non-Organic', 'Transitional', 'Organic']).defaultTo('Non-Organic');
-    t.dateTime('effective_date').notNullable().defaultTo(knex.fn.now());
-    t.boolean('deleted').defaultTo(false);
+    t.enu('organic_status', ['Non-Organic', 'Transitional', 'Organic']).defaultTo('Non-Organic');
+    t.date('effective_date').notNullable();
```

Comment:
At some point, we decided to use date instead of DateTime for date input by user